R/drive_upload.R

```diff
@@ -40,11 +41,33 @@
 #'
 #' ## clean-up
 #' drive_find("BioC_mirrors") %>% drive_rm()
+#'
+#' ## Upload a file with multiple parents
+#'
+#' ## Create mutltiple parent directories
+#' folder_1 <- drive_mkdir("folder_one")
+#' folder_2 <- drive_mkdir("folder_two")
+#'
+#' ## Upload the file into multiple parents. Notice here we are using
+#' ## the `addParents` parameter via the `...`.
+#' logo <- drive_upload(
+#'   R.home('doc/html/logo.jpg'),
+#'   folder_1,
+#'   addParents = folder_2$id
+#' )
+#'
+#' ## Examine the multiple parents
+#' drive_add_path(logo)
+#'
+#' ## Clean up
+#' bind_rows(folder_1, folder_2, logo) %>%
```

Comment:
You'll need to just do three separate `drive_rm()` calls. We don't want to use dplyr in our examples. It's only a suggested package. You could use a naming scheme that allows you to pipe the result of `drive_find()` into `drive_rm()`.